src/activities/quizzes/attempts/QuizAttemptsEntity.js

```diff
@@ -63,6 +63,16 @@ export class QuizAttemptsEntity extends Entity {
 		return entity.hasClass(Classes.quizzes.attempts.retakeIncorrectOnly) && entity.hasClass(Classes.quizzes.checked);
 	}
 
+	/**
+	 * @returns {object} an array of quiz attempt conditions sub-entities
+	 */
+	attemptConditions() {
+		const entity = this.getAttemptConditionsSubEntity();
+		if (!entity) return;
+		const attemptConditions = entity.getSubEntitysByClass(Classes.quizzes.attempts.attemptCondition);

```

Comment:
Should it be `getSubEntitiesByClass`?

Reply:
Weird that one should be as you commented.